docs/excel/make-your-excel-add-in-backwards-compatible-with-com-add-in-or-xll.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Make your Excel add-in backwards compatible with and existing COM add-in or Excel XLL
+description: 'Enable backwards compatibility with COM add-ins or Excell XLLs that have equivalent functionality to your Office web add-in'
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# Make your Excel add-in backwards compatible with and existing COM add-in or Excel XLL
+
+If you have existing COM add-ins or Excel XLL's, you can build equivalent Office web add-ins to extend your solution features to other platforms such as online or macOS. However Office web add-ins don't have all of the functionality available in COM add-ins and XLLs. Depending on the functionality your solution uses, users may have a better experience by using the original COM add-in or XLL when they use Excel on desktop.
+
+You can configure your Office web add-in so that when an equivalent COM add-in or XLL is already installed on the user's computer, Excel runs the COM add-in or XLL instead of your web add-in.
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+## Specify equivalent add-ins in the manifest
+
+To enable backwards compatibility, specify the equivalent add-ins in your manifest. You can specify a COM add-in, an XLL, or both. Then Excel will use the COM add-in or XLL instead of your web add-in when running on desktop.
+
+To set the equivalent COM add-in, you specify its `ProgID`. Then Excel will create the mapping between your COM add-in UI and your web add-in UI.
+
+To set the equivalent XLL for your custom functions, specify the `FileName` of the XLL. When the user opens a workbook with functions from the XLL, it will convert the functions to compatible functions. Then the workbook will use the XLL when opened in Excel on desktop, and it will use custom functions from your web add-in when opened online or on macOS.
```

Comment:
Technically it changes them on first open (if functions were XLL functions). Then the user saves. This is written directly to the file format on disk. Then when opened later, they are already compatible, so no more changes happen. Do you think this should be documented? It doesn't affect how you program anything so I didn't think it was necessary, but it might be interesting (can even show the text format it saves in.)

Reply:
Yeah, I guess isn't not necessary to document. Makes me wish you could add a footnote :-)